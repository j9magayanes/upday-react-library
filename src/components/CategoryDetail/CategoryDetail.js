import React from 'react';
import './categoryDetail.css';


export const CategoryDetail = ({ args }) => {

  return (
   <h1 className={args.appName==='meinestadt'?'categoryDetail meinestadt':'categoryDetail'}>{args.category}</h1>
  );
};