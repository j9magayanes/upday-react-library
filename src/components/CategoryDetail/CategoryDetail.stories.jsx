import React from 'react';

import { CategoryDetail } from './CategoryDetail';


export default {
  title: 'Form/CategoryDetail',
  component: CategoryDetail,
};

const Template = (args) =><CategoryDetail {...args}/>;


export const Default = Template.bind({});
Default.args = {
  label: 'Default',
 
 category: 'Default',
 args:{ appName: 'meinestadt',  category: 'Default',}

};

export const Meinestadt = Template.bind({});
Meinestadt.args = {
  label: 'Meinstadt',
  category: 'Top News',
  args:{ appName: 'meinestadt',  category: 'Default',}
};