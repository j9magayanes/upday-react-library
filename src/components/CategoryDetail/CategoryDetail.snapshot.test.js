import React from 'react';
import renderer from 'react-test-renderer';
import { CategoryDetail } from './CategoryDetail';

test('renders CategoryDetail component correctly', () => {
  const props = {
    category: 'Your Category',
  };

  const component = renderer.create(<CategoryDetail {...props} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
