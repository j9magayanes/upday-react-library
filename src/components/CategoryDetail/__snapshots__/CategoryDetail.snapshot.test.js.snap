// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CategoryDetail component correctly 1`] = `
<category-detail
  id="category-detail"
>
  Your Category
</category-detail>
`;
